packages/terra-action-header/src/ActionHeader.module.scss

```diff
@@ -1,56 +1,101 @@
   }
 
-  .right-buttons > * {
+  .right-buttons > button {
     margin-left: var(--terra-action-header-right-buttons-margin-left, 0.5rem);
   }
 
+  .header-button {
+    min-height: var(--terra-action-header-buttons-min-height, 1.429);
+    padding-bottom: var(--terra-action-header-buttons-padding-bottom, 0.286rem); // needed for alternate theme button overrides
+    padding-top: var(--terra-action-header-buttons-padding-top, 0.286rem); // needed for alternate theme button overrides
+    
+    > span {
+      line-height: var(--terra-action-header-buttons-inner-span-line-height); // needed for alternate theme button overrides
+      margin: var(--terra-action-header-buttons-inner-span-margin, 0 0.5rem); // needed for alternate theme button overrides
+    }
+  }
+
   .previous-next-button-group {
     display: inline-flex;
+    margin-left: var(--terra-action-header-previous-next-button-group-margin-left, 0.35714rem);
+    margin-right: var(--terra-action-header-previous-next-button-group-margin-right, 0);
     position: relative;
     vertical-align: middle;
     white-space: nowrap;
   }
 
-  .header-icon {
+  .back-button + .previous-next-button-group,
+  .maximize-button + .previous-next-button-group,
+  .minimize-button + .previous-next-button-group {
+    margin-left: var(--terra-action-header-buttons-plus-previous-next-buttons-margin-left, 0.5rem);
+  }
+
+  .header-button-icon {
     background-repeat: no-repeat;
     background-size: auto;
     display: inline-block;
+    height: var(--terra-action-header-button-icon-height, 1rem); // needed for different icon size in alternate themes
     position: relative;
-    top: var(--terra-action-header-icon-top, 0);
-    vertical-align: var(--terra-action-header-icon-vertical-align, -0.14285rem);
+    top: var(--terra-action-header-button-icon-top, 0);
+    vertical-align: var(--terra-action-header-button-icon-vertical-align, -0.14285rem);
+    width: var(--terra-action-header-button-icon-width, 1rem); // needed for different icon size in alternate themes
     @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
-      height: var(--terra-action-header-icon-ms-height, 1rem);
-      vertical-align: var(--terra-action-header-icon-ms-vertical-align, -0.17285rem);
+      height: var(--terra-action-header-button-icon-ms-height, 1rem);
+      vertical-align: var(--terra-action-header-button-icon-ms-vertical-align, -0.17285rem);
     }
 
     &.back {
       background-image: var(--terra-action-header-back-background-image, inline-svg('<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path d="M48 21H10.6L27.3 3.9 23.5.1 0 24l23.5 23.9 3.8-3.8L10.6 27H48z"></path></svg>'));
+
+      &:hover {
+        background-image: var(--terra-action-header-back-hover-background-image, inline-svg('<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path d="M48 21H10.6L27.3 3.9 23.5.1 0 24l23.5 23.9 3.8-3.8L10.6 27H48z"></path></svg>'));
```

Comment:
Why are we adding the same background image when the button is hovered?

Reply:
For themes that need a different hover color.